results.tex

```diff
@@ -0,0 +1,145 @@
+\section{Results}
+The results consist of the following plots:
```

Comment:
I assume this is just a placeholder and actual textual exposition will appear here in a final draft.